helpers/utilities.py

```diff
@@ -4,120 +4,183 @@
 from lemoncheesecake.matching import *
 import lemoncheesecake.api as lcc
 from selenium.webdriver.support.ui import Select
-from helpers.constants import *
-import requests
+from selenium.common.exceptions import TimeoutException, NoSuchElementException
 import random
 import string
 import time
 
 # now_time = datetime.utcnow()
 
-def click_element_by_xpath(driver, locator_value):
-    wait = WebDriverWait(driver, 15)
-    element = wait.until(
-        ec.presence_of_element_located((By.XPATH, locator_value)))
+def wait_for_element(driver, locator_type, locator_value):
+    try:
+        wait = WebDriverWait(driver, 30)
+        element = wait.until(
+            ec.visibility_of_element_located((locator_type, locator_value)))
+    except TimeoutException as e:
```

Comment:
maybe you want to add  an exception for staleElementException too rather than just timeout.